content/sensu-core/2.0/guides/influx-db-metric-handler.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "How to populate InfluxDB metrics using handlers"
+linkTitle: "Populating InfluxDB with Handlers"
+weight: 125
+version: "2.0"
+product: "Sensu Core"
+platformContent: false
+menu:
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What are Sensu handlers?
+
+Sensu event handlers are actions executed by the Sensu server on [events][1].
```

Comment:
I'm wondering if we could figure out a way to avoid duplicating this content from https://docs.sensu.io/sensu-core/2.0/guides/send-slack-alerts/ 🤔

Reply:
I wasn't sure if the brief intro on handlers would be beneficial here. Maybe instead of those questions we could say something along the lines of "do you know what handlers are? If not, go check out the handlers guide"?

Reply:
👍👍👍